docs/community/glossary.md

```diff
@@ -0,0 +1,56 @@
+---
+id: glossary
+title: 'OpenSauced Glossary'
+sidebar_label: OpenSauced Glossary
+keywords:
+  - glossary
+---
+
+Welcome to the glossary, where we define concepts used on our platform to help you get the most out of OpenSauced.
+
+## PR Velocity
+
+PR Velocity measures the percentage and amount of days that pull requests that are merged. It is a great way to track the progress of your open-source contributions.
+
+## Activity 
+
+Activity tracks PRs over the last 30 days. High indicates more than 80 PRs. Medium is between 5 and 80 PRs. Low is below 5 PRs.
```

Comment:
You can delete this first sentence since you repeat the information below with the list, which is easier to read.

Reply:
Hey @Lymah123 
You haven't deleted the first sentence here. :)

Reply:
> Hey @Lymah123 You haven't deleted the first sentence here. :)

Is it okay like that at @adiati98 ?